.github/workflows/daily.yml

```diff
@@ -69,12 +69,21 @@ jobs:
       - name: Run stubtest
         shell: bash
         run: |
+          PACKAGES=$(python tests/get_packages.py)
+
           if [ "${{ matrix.os }}" = "ubuntu-latest" ]; then
-            sudo apt update
-            sudo apt install -y $(python tests/get_packages.py)
+            sudo apt update && sudo apt install -y $PACKAGES
 
             xvfb-run python tests/stubtest_third_party.py --specified-stubs-only --num-shards 4 --shard-index ${{ matrix.shard-index }}
           else
+            if [ "${{ matrix.os }}" = "macos-latest" ]; then
+              NONINTERACTIVE=1 brew install $PACKAGES
+            fi
+
+            if [ "${{ matrix.os }}" = "windows-latest" ]; then
```

Comment:
```suggestion
            if [ ! -z "$PACKAGES" ] && [ "${{ matrix.os }}" = "windows-latest" ]; then
```

Reply:
@AlexWaygood can you please apply this change? :)

This will fix the CI and we can merge it.

Reply:
It's probably worth making this change to the `apt_dependencies` and `brew_dependencies` branches of this script as well, right? What if we remove stubs for `JACK-Client` and `pyaudio` at some point in the future, and then the CI randomly starts failing because suddenly we're passing in 0 packages to the `brew install` part of this workflow?

Anyway, is there a massive rush? It's @Avasam's PR :)

Reply:
No rush on my side. I simply wanted to complete all the stubs that benefited from sobolevn's work with multi-platform support. And this is the last PR in that series.

Whilst more unlikely, I've added the check for apt installs as well. It doesn't hurt.

I've also changed the os condition to use `runner.os` instead of `matrix.os`. This way the exact os version can be updated without braking the condition.